apps/clkinfomsg/clkinfo.js

```diff
@@ -0,0 +1,88 @@
+(function() {
+  
+  var unreadImg = function() {
+    return atob("GBiBAAAAAAAAAAAAAB//+D///D///D///D///D///D///D5mfD5mfD///D///D///D///D///D///B//+APgAAOAAAOAAAAAAAAAAA==");
+  }
+  var allImg = function() {
+    return atob("GBiBAAAAAAAAAAB+AAD/AAPDwA8A8B4AeDgAHDgAHDwAPD8A/D/D/D/n/D///D///D///D///D///D///B//+AAAAAAAAAAAAAAAAA==");
+  }
+
+  var debug = function(o) {
+    //console.log(o);
+  }
+  var msgUnread;
+  var msgAll;
+  var msgs = require("messages");
+
+  var getAllMGS = function() {
```

Comment:
Minor typo - should this be `getAllMSGs` ?

Reply:
Good catch - fixing